assessments/FastBridge/templates/studentAssessments.jsont

```diff
@@ -0,0 +1,339 @@
+{
+  "studentAssessmentIdentifier": "{{student_assessment_identifier}}",
+  "assessmentReference": {
+    "assessmentIdentifier": "{{assessmentIdentifier}}",
+    "namespace": "{{assessmentIdentifier_namespace}}"
+  },
+  "reportedSchoolReference": {
+    "schoolId": {{School}}
+  },
+  "schoolYearTypeReference": {
+    "schoolYear": {{SchoolYear}}
+  },
+  "studentReference": {
+    "studentUniqueId": "{{student_unique_id}}"
+  },
+  "administrationLanguageDescriptor": "uri://ed.sc.gov/LanguageDescriptor#{{AssessmentLanguageDescriptor}}",
+  "period": {
+    "assessmentPeriodDescriptor": "{{assessment_period_namespace}}#{{assessment_period_codeValue}}"
+  },
+  {#-----------------------------------------------------------------------------------------------------------#}
```

Comment:
do comments like this not break the output?

Reply:
No, they are ignored by the renderer.